content/blog/2024-01-06-TidyTuesday/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Bring Your Own Data week for TidyTuesday
+author: Yanina Bellini Saibene
+summary: "It's _Bring Your Own Data_ week for _#TidyTuesday_, and I decided to analyze my records of events."
+date: '2024-01-06'
+categories:
+  - 100DaysToOffload
+  - English
+tags:
+  - English
+  - 100DaysToOffload
+  - rstats
+  - TidyTuesday 
+---
+
+It's _Bring Your Own Data_ week for _#TidyTuesday_, and I decided to analyze my records of events. 
+
+I have the habit of recording everything because of my academic jobs. I continue to write down the different events in which I participate and some interesting information about them.  In addition to update the data with what I did during 2023, I have been cleaning up the data, completing the information and removing duplicates.
```

Comment:
```suggestion
I have the habit of recording everything because of my academic jobs. I continue to write down the events I participate in and some interesting information about them.  For this exercise, in addition to updating the data with what I did during 2023, I have been cleaning up the data, completing the information, and removing duplicates.
```